sdks/java/io/kinesis/pom.xml

```diff
@@ -160,6 +146,12 @@
     </dependency>
 
     <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava-testlib</artifactId>
```

Comment:
A while ago we debated whether to even have a test dep on `guava-testlib`. I was for it, since we could really use `EqualityTester` everywhere, but the decision ended up against it. So I'm going to say LGTM since time has passed, and we can rekindle objections but should just solve them with shading.